libs/category/testing/src/inmemory-backend/category.service.ts

```diff
@@ -17,6 +18,8 @@ export class DaffInMemoryBackendCategoryService implements InMemoryDbService {
 
   constructor(private categoryFactory: DaffCategoryFactory) {
     this.category = this.categoryFactory.create();
+
+    this.category.productIds = IN_MEMORY_PRODUCT_IDS.slice(0, Math.floor(Math.random() * 8 + 1));
```

Comment:
Could we instead dependency inject the current product in-memory service and map over its products instead of a hard reference like this?

Reply:
That makes sense. Done